packages/genesys-spark-components/src/components/stable/gux-rating/gux-rating.tsx

```diff
@@ -28,6 +28,9 @@ export class GuxRating {
   @Prop()
   readonly: boolean = false;
 
+  @Prop()
+  increment: 0.5 | 1 = 1;
```

Comment:
This attribute/property can be set to any number. If you really want to support only these two options, it might be best to move to two specific string values instead (default and halfStep?).

Reply:
What do you mean "can be set to any number"? You will get a type error if you set it to something other than `0.5` or `1`.

Reply:
The TS types don't really exist once this gets transpiled to JS. JS only has number, so that is what we get. I tested both the attribute and property in the demo and was able to set it to a variety of other numbers (.25. 1.5, 2, 3). I can even set it to `steve` without getting a type error (I think it is just getting coerced to `0` based on behavior).

Reply:
I agree with Thomas, I think it might would be better to set the increment as a string. `default` and `halfStep` sound good to me

Reply:
Done